Tetris/src/tetris/Tetromino.java

```diff
@@ -2,8 +2,83 @@
 
 /**
  * Template for tetromino characteristics and creation.
+ * 
+ * Contains the components necessary for a tetromino to exist within the game
+ * Minimal logic to implement selection of tetromino shape upon creation
  *
  */
 public class Tetromino {
-
+	
+	private boolean[][] shape; //this may be implemented in a different way
+						// but it must match the return type of getShape()
+	
+	private int x_location;
+	private int y_location;
+	
+	/**
+	 * Constructor for tetromino
+	 * Grabs shape character and initializes shape array based on selection
+	 * @param shape
+	 * char shape may be 'i', 'o', 's', 'z', j', or 'l'
+	 */
+	public Tetromino(char shape){
+		//if shape == 'l' shape ==  oooo
+		
+		//else if shape == 'o' shape == oo
+		//                              oo
+		
+		//etc
+	}
+	
+	/**
+	 * Retriever for shape data
+	 * Used by system and display
+	 * @return shape
+	 */
+	public boolean[][] getShape(){
+		return this.shape;
+	}
+	
+	/**
+	 * Getter for x location
+	 * X position is specified as:
+	 * TODO: decide how
+	 * @return this.x_location
+	 */
+	public int getXLocation(){
+		return -1;
+	}
+	
+	/**
+	 * Getter for y location
+	 * Y position is specified as:
+	 * TODO: decide how
+	 * @return this.y_location
+	 */
+	public int getYLocation(){
+		return -1;
+	}
+	/**
+	 * Set location of tetromino object
```

Comment:
We need to talk about exactly what X and Y we will be referencing. Example: Upper left corner of the tetromino's grid?